### PR TITLE
feat(ci): revolutionize repo with github actions ✨

### PR DESCRIPTION
**Pull request by Hazmi35**

### CODE REVIEW

**Comment by Hazmi35** [comment]
Please note this PR are unfinished, I'm still dealing with conflicts with @KurokuTetsuya

**Review by mzrtamp — APPROVED** [review]

**Comment by Hazmi35** [comment]
this pull request is a part of #5, please merge it before merging this pull request, also please add NPM_TOKEN in secret from the settings page

**Comment by Hazmi35** [comment]
#5 merged, please add NPM_TOKEN secret and merge this when you're ready

**Pull request merged**
